lib/salestation/web/responses.rb

```diff
@@ -13,6 +13,10 @@ def self.to_ok
         -> (object) { Deterministic::Result::Success(Responses::OK.new(body: object)) }
       end
 
+      def self.to_no_content
+        ->  (*) { Deterministic::Result::Success(Responses::NoContent.new(body: {})) }
```

Comment:
human lint: two spaces between -> and (

Reply:
Good eye https://github.com/salemove/salestation/pull/20/commits/5724141ded08d229c32fb915e91a4fdb7fb0e86c

There are bunch of rubocop errors in this project. Need to be fixed in a separate PR.